web/migrations/0054_badge_custom_type_peermessage_encrypted_key_and_more.py

```
@@ -0,0 +1,155 @@
+
+    dependencies = [
+        ("web", "0053_goods_featured"),
+        migrations.swappable_dependency(settings.AUTH_USER_MODEL),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name="badge",
+            name="custom_type",
+            field=models.CharField(
+                blank=True, help_text="Custom badge type (optional, overrides predefined types)", max_length=100
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)